packages/explat-client/src/internal/experiment-assignment-store.ts

```diff
@@ -1,10 +1,11 @@
+import * as ExperimentAssignments from './experiment-assignments';
 import localStorage from './local-storage';
 import * as Validations from './validations';
 import type { ExperimentAssignment } from '../types';
 
-const localStorageExperimentAssignmentKeyPrefix = 'explat-experiment-';
+export const localStorageExperimentAssignmentKeyPrefix = 'explat-experiment-';
 
-const localStorageExperimentAssignmentKey = ( experimentName: string ): string =>
+export const localStorageExperimentAssignmentKey = ( experimentName: string ): string =>
```

Comment:
Let's mention that these are exported only for testing (including all the helper functions below.)

If they are being exported only for testing and are simple functions they also don't need documentation generally.

Reply:
Sounds good! Ironically, I actually had it commented like that at first and then second guessed myself and deleted all those comments at the last minute before committing 😅😂